src/OceanSeaIceModels/PrescribedAtmospheres.jl

```diff
@@ -102,7 +102,7 @@ const CP{FT} = ConstitutiveParameters{FT} where FT
 @inline gas_constant(p::CP)   = p.gas_constant
 @inline molmass_dryair(p::CP) = p.dry_air_molar_mass
 @inline molmass_water(p::CP)  = p.water_molar_mass
-@inline molmass_ratio(p::CP)  = molmass_dryair(p) / molmass_water(p)
+@inline Rv_over_Rd(p::CP)     = molmass_dryair(p) / molmass_water(p)
 @inline R_v(p::CP)            = gas_constant(p) / molmass_water(p)
 @inline R_d(p::CP)            = gas_constant(p) / molmass_dryair(p)
```

Comment:
R_v and R_d are in parameters now rather than derived in thermodynamics, so I would just use them directly.

Reply:
are you suggesting

```Julia
@inline Rv_over_Rd(p::CP)     = molmass_dryair(p) / molmass_water(p)
@inline R_v(p::CP)            = gas_constant(p) / molmass_water(p)
@inline R_d(p::CP)            = gas_constant(p) / molmass_dryair(p)
```

to be replaced by

```Julia
@inline Rv_over_Rd(p::CP)     = p.Rv_over_Rd
@inline R_v(p::CP)            = p.R_v
@inline R_d(p::CP)            = p.R_d
```
?

Reply:
Yes.